widget/embedded/src/store/utils/data.ts

```diff
@@ -115,3 +119,16 @@ export function matchTokensFromConfigWithMeta(params: {
 
   return Object.values(result);
 }
+
+// Get similar blockchains between 'from' and 'to' configurations
+export function getSupportedBlockchainsFromConfig(params: {
+  config: WidgetConfig;
+}): string[] {
+  const { config } = params;
+  const configFromBlockchains = config.from?.blockchains || [];
+  const configToBlockchains = config.to?.blockchains || [];
+
+  return configFromBlockchains.filter((type) =>
```

Comment:
You can use `Set` to have unique values.

Reply:
I used set for this

Reply:
Regarding you change, I think this is redundant and can be removed.
```
  if (!configFromBlockchains.length || !configToBlockchains.length) {
    return [];
  }
```